src/main/fc/rc.c

```diff
@@ -67,11 +67,14 @@ static float maxRcDeflectionAbs;
 static bool reverseMotors = false;
 static applyRatesFn *applyRates;
 
-static uint16_t currentRxIntervalUs; // packet interval in microseconds
-static float currentRxRateHz; // packet rate in hertz
+#define RX_INTERVAL_MIN_US     950 // 0.950ms to fit 1kHz without an issue
+#define RX_INTERVAL_MAX_US   65500 // 65.5ms or 15.26hz
```

Comment:
These two defines are easier to identify by putting them at the top in line 58 (so they are not "needles in a haystack" / to separate them from variable declarations)

Reply:
thanks, done